namespaces/live.cloud-platform.service.justice.gov.uk/parliamentary-questions-development/resources/ecr.tf

```diff
@@ -7,11 +7,8 @@ module "pq_ecr_credentials" {
 
   repo_name = var.repo_name
 
-  # enable the oidc implementation for CircleCI
-  oidc_providers = ["circleci"]
-
-  # specify which GitHub repository your CircleCI job runs from
-  github_repositories = [var.repo_name]
+  oidc_providers      = ["github"]
```

Comment:
The name implies that you can have multiple providers so that might be the way to go while we are transitioning?